www/doc/1.7/report.html

```diff
@@ -20,143 +22,176 @@ <h1>Herbie reports</h1>
+  <h2 id="initial">Initial Program</h2>
 
-  <p>Fifth, Herbie's derivation of its output program. These can be
-    helpful in understanding how Herbie works. Each substantive step
-    in the derivation also lists the error, in bits, of that step's
-    output. Sometimes you can use that to pick a less-complex and
-    not-substantially-less-accurate program.</p>
-
-  <p>Derivations sometime name arithmetic laws used by Herbie, or they
-    might claim derivation steps are done by simplification, series
-    expansion, or other Herbie strategies. The derivation will also
-    call out any time the input is split into cases. When one part of
-    the program is colored blue, that is the part modified in that
-    derivation step.
+  <p>Fifth, The initial program given as the baseline for herbie to improve on.
+    Showing the initial Accuracy and marking the speed as 1.0x.
   </p>
+  <figure>
+    <img width="100%" src="initial-program.png" />
+    <figcaption>
+      Displays the initial Program and accuracy for the range of inputs.
+    </figcaption>
+  </figure>
 
+  <h2 id="alternative">Alternatives</h2>
   <figure>
-    <img width="100%" src="report-derivation.png" />
-    <figcaption>A short derivation from a Herbie report. Note the
-    error, in gray, measured after each derivation step.</figcaption>
+
+    <img width="100%" src="alternative.png" />
+    <figcaption>
+      A display of a given alterative found by Herbie. Along the top accuracy
+      of sampled points and speed up relative to the original program are given.
+      Along with a link to the <a href="#derivation">Derivation</a> of the alterative program.
+    </figcaption>
   </figure>
 
-  <h2 id="reproduction">Reproduction</h2>
 
-  <p>Sixth, a command to reproduce this Herbie result. If you find a
-  Herbie bug, include this code snippet when
-  <a href="https://github.com/uwplse/herbie/issues">filing an
-  issue</a>.</p>
+  <h3 id="derivation">Derivation</h2>
 
-  <figure>
-    <img width="100%" src="report-reproduce.png" />
-    <figcaption>Reproduction information for a Herbie run. You can use
-    this when submitting bug reports.</figcaption>
-  </figure>
+    <p>Herbie's derivation of its output program. These can be
+      helpful in understanding how Herbie works. Each substantive step
+      in the derivation also lists the accuracy of that step's
+      output. Sometimes you can use that to pick a less-complex and
+      not-substantially-less-accurate program.</p>
 
-  <h2 id="links">Additional links</h2>
-  
-  <p>The top of the page has a right-hand menu bar with additional
-  links. “Report” returns you to Herbie's main page. “Log” and
-  “Metrics” give you detailed internal information about Herbie.</p>
+    <p>Under <em>Step-by-step derivation</em> Herbie sometimes names arithmetic
+      laws used, or they might claim derivation steps are done by
+      simplification, series expansion, or other Herbie strategies. The
```

Comment:
```suggestion
      laws used, or may say <kbd>metadata-compute</kbd> if Herbie used direct computation in a particular step.
```
"Step by step derivation' never says simplification or series expansion or anything else.